js/base/Exchange.js

```diff
@@ -3159,4 +3159,25 @@ module.exports = class Exchange {
         }
         return fee;
     }
+
+    parseIncomes (incomes, market = undefined, since = undefined, limit = undefined) {
```

Comment:
we have other exchanges with this implementation, should we replace it there as well?
![image](https://user-images.githubusercontent.com/43336371/217068685-aea967a7-b456-4599-a715-6c8d99a975d3.png)

Reply:
yeah

Reply:
I opened a position on huobi so that I could test it out in a few hours, I dont have apiKeys for woo though, could you test that one out?

Reply:
@samgermain probably not directly related to these changes but something is off, check this out:
```
p huobi fetchFundingHistory "LTC/USDT:USDT" None 1          
Python v3.10.9
CCXT v2.7.44
huobi.fetchFundingHistory(LTC/USDT:USDT,None,1)
[{'amount': 0.000382339,
  'code': 'USDT',
  'datetime': '2023-02-07T00:00:23.206Z',
  'id': '1292654517',
  'info': {'amount': '0.000382339000000000',
           'asset': 'USDT',
           'contract_code': 'ADA-USDT',
           'face_margin_account': '',
           'id': '1292654517',
           'margin_account': 'USDT',
           'query_id': '1292654517',
           'trade_partition': 'USDT',
           'ts': '1675728023206',
           'type': '30'},
  'symbol': 'ADA/USDT:USDT',
  'timestamp': 1675728023206}]
```
⬆️ requested LTC/USDT:USDT, got ADA/USDT:USDT

Reply:
@carlosmiei You're right, it's not related, it happens on master as well because the id isn't being sent to the exchange, and we're not filtering by symbol.

- should we filter by symbol?
- I'll post another PR if it's possible to send the id to the exchange, I'm guessing that we don't send it because we're not able to though

Reply:
@samgermain 
> should we filter by symbol?
Yeah I think so

Reply:
Will merge this PR then, don't forget to open the other one with the fix pls

Reply:
> Will merge this PR then, don't forget to open the other one with the fix pls

I won't, it just lead to a chain of other fixes I needed to make